compiler/vm.nim

```diff
@@ -692,6 +692,9 @@ proc rawExecute(c: PCtx, start: int, tos: PStackFrame): TFullReg =
       regs[ra].intVal =
         ord(exprStructuralEquivalent(regs[rb].node, regs[rc].node,
                                      strictSymEquality=true))
+    of opcEqNodeType:
+      decodeBC(rkInt)
+      regs[ra].intVal = ord(regs[rb].node.typ == regs[rc].node.typ)
```

Comment:
Then you can compare the string representations of getType. That said, I don't think there is a single "identicalType" comparison in the whole compiler, so I don't think there are valid use cases for this.